cmd/asset-syncer/utils.go

```diff
@@ -325,6 +391,10 @@ func extractFilesFromBuffer(buf *bytes.Buffer) (*artifactFiles, error) {
 		}
 
 		compressedFileName := header.Name
+		if len(strings.Split(compressedFileName, "/")) > 2 {
```

Comment:
Might be more straight forward to use `filepath.Dir(compressedFileName) != "/"` or similar?

Reply:
for doing so, we need the name of the app, it would be something like `filepath.Dir(compressedFileName) != appName` but I don't have that name in this function (and I don't think it's worth it to add it just for this).

Reply:
Right, OK - I misread and thought the path was just `/Readme.md`. You could still use https://golang.org/pkg/path/filepath/#SplitList (which is effectively the same as your code, just using the filepath library which is OS independent for slashes etc., but not so relevant here, I agree). All good :)